cobc/cobc.c

```diff
@@ -3905,6 +3906,17 @@ process_command_line (const int argc, char **argv)
 		cobc_early_exit (EXIT_SUCCESS);
 	}
 
+	/* Load the collating tables */
+	if (cb_ebcdic_table == NULL) {
+		cb_ebcdic_table = cobc_main_strdup (
+			cb_flag_alt_ebcdic ? "alternate" : "default");
+	}
+	if (cob_load_collation (cb_ebcdic_table,
```

Comment:
Please postpone this call until we know we need it, when `cb_ebcdic_table == NULL`.

Reply:
You mean, calling cob_load_collation in `codegen.c:output_collating_tables` ?

Reply:
Yes, at least if there was no explicit request up-front. Otherwise we'd read in these files on _any_ call of cobc.